<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-7SBGRNK3" ignored="0" name="Test1" tests="6" failures="3" timestamp="30 Mar 2020 19:39:53 GMT" time="97.027" errors="0">
  <testcase name="loginTest" time="11.827" classname="com.realtimeProject.testcases.TC_LoginTest_001"/>
  <testcase name="loginTest" time="21.766" classname="com.realtimeProject.testcases.TC_LoginDDT_002"/>
  <testcase name="loginTest" time="12.313" classname="com.realtimeProject.testcases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.realtimeProject.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:45)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="11.935" classname="com.realtimeProject.testcases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.realtimeProject.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:45)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="12.231" classname="com.realtimeProject.testcases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.realtimeProject.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:45)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="addNewCustomer" time="12.852" classname="com.realtimeProject.testcases.TC_AddCustomerTest_003"/>
</testsuite> <!-- Test1 -->
